swagger/src/main/scala/org/scalatra/swagger/Swagger.scala

```diff
@@ -338,13 +338,12 @@ object DataType {
     else if (isDateTime(klass)) this.DateTime
     else if (isBool(klass)) this.Boolean
     //    else if (classOf[java.lang.Enum[_]].isAssignableFrom(klass)) this.Enum
-    //    else if (isMap(klass)) {
```

Comment:
wait, the code was there but commented out?

Reply:
It was, but I didn't yet found why. Maybe it's because scalatra wasn't ready to support them in Swagger 1.0

Reply:
https://github.com/scalatra/scalatra/issues/678